_config.yml

```diff
@@ -48,7 +48,7 @@ exclude: ["downloads/RELEASE_NOTE.md","vendor","0.991/","v1-2","v1-1","v1-0","v0
 #   - vendor/cache/
 #   - vendor/gems/
 #   - vendor/ruby/
-dist_server: https://dist-dev.ballerina.io
+dist_server: https://dist.ballerina.io
```

Comment:
This needs to be kept different from prod-sync to master

Reply:
I see. So basically the 2 branches have diverged now? 🤔